data-access/src/functions/http-graphql/schema/types/community.graphql

```diff
@@ -13,8 +13,38 @@ type Community implements MongoBase {
   createdAt: DateTime
   updatedAt: DateTime
   userIsAdmin: Boolean
+  approvedVendors: [VendorUser]
 }
 
+type VendorUserPersonalInformation {
```

Comment:
Remove all of these VendorUser types, we already defined everything we need for VendorUser under the vendor-user.graphql file. Graphql will be able to use types across files, we don't need to redefine everything here locally